src/desktop/IPC.js

```diff
@@ -96,14 +103,24 @@ export class IPC {
 						d.reject(e)
 					})
 				break
+			case 'integrateDesktop':
```

Comment:
Should these wait for the `integrate()` so that web client gets an error if it fails?